ui.apps/src/main/content/jcr_root/apps/acs-commons/granite/ui/components/include/include.jsp

```diff
@@ -0,0 +1,26 @@
+<%@ include file="/libs/granite/ui/global.jsp" %><%
```

Comment:
This might be a dumb question, but is there a way to *not* use a JSP here? Can this just be Java code? That would obviate the need (I think) for the `NamespacedTransformedResourceProvider` interface to be exported.

Reply:
Excellent point. Il give it a try and let you know.

Reply:
Excellent point. Il give it a try and let you know.

Reply:
Ok I really tried with a sling GET servlet, but this introduces a lot of complexity and I didn't get it to work. PR stands as is.

Reply:
thanks for trying :)

Reply:
Can you look at the 1 remainder point from @davidjgonzalez above? :D